tests/solvers/nn/icnn_test.py

```diff
@@ -21,7 +21,7 @@
 @pytest.mark.fast()
 class TestICNN:
 
-  def test_icnn_convexity(self, rng: jax.random.PRNGKeyArray):
+  def test_icnn_convexity(self, rng: jax.random.PRNGKey):
```

Comment:
Can you please revert this? It should be `PRNGKeyArray`, I made a typo in the skeleton.